### DISCUSSION TITLE
Is there a way to know TDK enabled CarPlay?

### DISCUSSION THREAD

**Discussion by 84798012** [smartdevicelink/sdl_ios#2096] [Q&A] [answered]
If an app runs in SDL and CarPlay together, there may caused unexpected issue. 
So I need to know whether user have enabled CarPlay, and give user prompt like this. (This from other app's)
And more, I want to know how to implement when tap "setting", then can goes "setting". Thanks!

Thanks
![IMG_6381](https://user-images.githubusercontent.com/3917323/174539093-e4f0a12c-6b8c-4cf9-9583-a752059ab803.jpg)
!

**Comment by joeljfischer** [comment, marked as answer]
> If an app runs in SDL and CarPlay together, there may caused unexpected issue.
> So I need to know whether user have enabled CarPlay, and give user prompt like this. (This from other app's)

This prompt is automatically given by the system, not an app. There is no SDL built-in way to detect when your app is unavailable to open like this that I'm aware of.

**Reply by 84798012** [reply]
Thanks for your help, then I'll ask TDK team how to handle it.
